packages/backend/src/app/helper/encryption.ts

```diff
@@ -0,0 +1,32 @@
+import * as crypto from 'crypto';
+import { system } from './system/system';
+import { SystemProp } from './system/system-prop';
+
+const encryptionKey = system.getOrThrow(SystemProp.ENCRYPTION_KEY);
+
+export interface EncryptedObject {
+    iv: string;
+    data: string;
+}
+
+export function encryptObject(object: unknown): EncryptedObject {
+    const iv = crypto.randomBytes(16); // Generate a random initialization vector
+    const key = Buffer.from(encryptionKey.padEnd(32, '\0'), 'binary'); // Create a key from the encryption key
```

Comment:
I think padding the encryption key with nulls is a bad idea. This could allow the key to be an empty string which kind of defeats the purpose of encryption.

A better option in my opinion is to validate that the key's length is 256 bits at the start and throw if this is not the case.